src/elements/content-sharing/ContentSharing.js

```diff
@@ -110,30 +145,122 @@ function ContentSharing({ apiHost, displayInModal, itemID, itemType, language, t
         }
     }, [api, getError, item, itemID, itemType, sharedLink, currentUserID]);
 
-    if (errorMessage) {
-        return <ErrorMask errorHeader={<FormattedMessage {...errorMessage} />} />;
+    // Close a notification
+    const handleNotificationClose = React.useCallback(
+        (id: number) => {
+            const updatedNotifications = { ...notifications };
```

Comment:
`{ ...notifications }` gives shallow copy. Intentional?

Reply:
Yup, `notifications` is a one-level-deep object (`{ id: <Notification> }`).